app/components/pair-compare.jsx

```diff
@@ -175,9 +175,9 @@ function PairCompare(props) {
 
                 <div className={classes.visiblePointsContainer}>
                     {idxLeft < pointList.length &&
-                        <div className={classes.visiblePoint} ref={visibleLeftPointRef} onClick={handleLeftPointClick}>{<Point {...pointList[idxLeft]} />}</div>}
+                        <div className={classes.visiblePoint} ref={visibleLeftPointRef} onClick={handleLeftPointClick} tabIndex={0} title={`Choose as more important: ${pointList[idxLeft]?.subject}`}>{<Point {...pointList[idxLeft]} />}</div>}
```

Comment:
To be able to use space-bar to 'click' the point, the point needs to be wrapped in a `<button>` rather than a div.  You can just change it and it will work, but you will have to override some of the button formatting in the class to make it look right.